### ISSUE TITLE
'hw mfu autopwn' 

### ISSUE THREAD

**Issue by iceman1001** [RfidResearchGroup/proxmark3#883]
The idea of checking pwds,  and then dump card in one swift command.

**Comment by mwalker33** [comment]
As the current hf mf autopwn seems to check the passwords and dump the card in one command, is there anything still outstanding on this  issue ?

**Comment by iceman1001** [comment]
yes...  a complete new command....

**Comment by iceman1001** [comment]
yeah,  we have some keygens,  some dictionary,   some extract pwd / pack from trace,
